diff -uraN jack2-1.9.21/jack.pc.in omv-jack2-1.9.21/jack.pc.in
--- jack2-1.9.21/jack.pc.in	2022-04-15 21:14:06.000000000 +0200
+++ omv-jack2-1.9.21/jack.pc.in	2022-05-02 20:51:01.192276391 +0200
@@ -9,4 +9,4 @@
 Description: the Jack Audio Connection Kit: a low-latency synchronous callback-based media server
 Version: @JACK_VERSION@
 Libs: -L${libdir} -l@CLIENTLIB@
-Cflags: -I${includedir}
+Cflags: -I@INCLUDEDIR@ -I${includedir}/jack
